package.json

```diff
@@ -2,42 +2,66 @@
   "name": "@google-cloud/trace-agent",
   "version": "2.1.3",
   "description": "Node.js Support for StackDriver Trace",
-  "main": "index.js",
-  "repository": "GoogleCloudPlatform/cloud-trace-nodejs",
+  "main": "build/index.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/GoogleCloudPlatform/cloud-trace-nodejs.git"
```

Comment:
Any reason for this change? The previous terse format looks easier to understand to me.

Reply:
Inadvertant from `gts init` - reverted